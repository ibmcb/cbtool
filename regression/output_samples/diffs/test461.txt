 status: VMC B7C08E2C-9D13-50D3-9838-9CB8CD513270 was successfully registered on SimCloud "TESTCLOUD".
 status: VMC B61C74E8-3B3D-5595-8E1B-35736A61345B was successfully registered on SimCloud "TESTCLOUD".
 status: VMC AA4F8ACC-E8A4-5354-8CE0-80BB6D2A0A81 was successfully registered on SimCloud "TESTCLOUD".
 status: VMC EAB78AD4-258F-52DD-B4B9-B058DF4FE365 was successfully registered on SimCloud "TESTCLOUD".
|simzone_a               |5               |sut                     |simzone_a                       |11.35.98.226     
|simzone_b               |5               |sut                     |simzone_b                       |219.1.130.98     
|simzone_c               |5               |sut                     |simzone_c                       |73.250.11.31     
|simzone_d               |5               |lg                      |simzone_d                       |96.37.237.88     
|Used Memory                                          |19.25M                        
|Uptime (in seconds)                                  |2517109                       
|Total Connections Received                           |185724                        
|Total Commands Processed                             |11809775                      
|Storage Size                                         |18292736                      
|Data Size                                            |6883636                       
|Index Size                                           |2142112                       
|Average Object Size                                  |614.007314245                 
|B7C08E2C-9D13-50D3-9838-9CB8CD513270 (simzone_c) VM RESERVATIONS|0                             
|EAB78AD4-258F-52DD-B4B9-B058DF4FE365 (simzone_a) VM RESERVATIONS|0                             
|B61C74E8-3B3D-5595-8E1B-35736A61345B (simzone_d) VM RESERVATIONS|0                             
|AA4F8ACC-E8A4-5354-8CE0-80BB6D2A0A81 (simzone_b) VM RESERVATIONS|0                             
