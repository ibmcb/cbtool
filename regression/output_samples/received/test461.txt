 status: VMC B7C08E2C-9D13-50D3-9838-9CB8CD513270 was successfully registered on SimCloud "TESTCLOUD".
 status: VMC B61C74E8-3B3D-5595-8E1B-35736A61345B was successfully registered on SimCloud "TESTCLOUD".
 status: VMC AA4F8ACC-E8A4-5354-8CE0-80BB6D2A0A81 was successfully registered on SimCloud "TESTCLOUD".
 status: VMC EAB78AD4-258F-52DD-B4B9-B058DF4FE365 was successfully registered on SimCloud "TESTCLOUD".
 status: Attribute "collect_from_host" was set to "false". Skipping Host OS performance monitor daemon startup
All VMCs successfully attached to this experiment.
The following VMCs are attached to this experiment (Cloud TESTCLOUD) :
|name                    |host_count      |pool                    |cloud_hostname                  |cloud_ip         
|simzone_a               |5               |sut                     |simzone_a                       |11.35.98.226     
|simzone_b               |5               |sut                     |simzone_b                       |219.1.130.98     
|simzone_c               |5               |sut                     |simzone_c                       |73.250.11.31     
|simzone_d               |5               |lg                      |simzone_d                       |96.37.237.88     

The VMC object simzone_c, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (VMC object key)               |value                                
|update_attempts                          |312                                  
|update_frequency                         |12                                   

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |19.25M                        
|Redis Version                                        |2.6.2                         
|Uptime (in seconds)                                  |2517109                       
|Total Connections Received                           |185724                        
|Total Commands Processed                             |11809775                      
|Number of Keys                                       |272                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.5                         
|Storage Size                                         |18292736                      
|Data Size                                            |6883636                       
|Index Size                                           |2142112                       
|Average Object Size                                  |614.007314245                 
|Collections                                          |13                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |0                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |12                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |8                             
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |0                             
|VMs ARRIVED                                          |0                             
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |0                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |60                            

 ---------------- VMC-WIDE COUNTERS ----------------
|B7C08E2C-9D13-50D3-9838-9CB8CD513270 (simzone_c) VM RESERVATIONS|0                             
|EAB78AD4-258F-52DD-B4B9-B058DF4FE365 (simzone_a) VM RESERVATIONS|0                             
|B61C74E8-3B3D-5595-8E1B-35736A61345B (simzone_d) VM RESERVATIONS|0                             
|AA4F8ACC-E8A4-5354-8CE0-80BB6D2A0A81 (simzone_b) VM RESERVATIONS|0                             

