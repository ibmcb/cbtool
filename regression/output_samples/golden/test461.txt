 status: VMC 851FD80B-471A-5915-8AE1-DEFC6E0B82CF was successfully registered on SimCloud "TESTCLOUD".
 status: VMC DA279768-EC83-57F0-909E-7559192E2C87 was successfully registered on SimCloud "TESTCLOUD".
 status: VMC 893C8680-1F87-5029-9204-9C8BBE37D21C was successfully registered on SimCloud "TESTCLOUD".
 status: VMC 322AE863-21B7-54F7-A0C4-7176A0E34BBF was successfully registered on SimCloud "TESTCLOUD".
 status: Attribute "collect_from_host" was set to "false". Skipping Host OS performance monitor daemon startup
All VMCs successfully attached to this experiment.
The following VMCs are attached to this experiment (Cloud TESTCLOUD) :
|name                    |host_count      |pool                    |cloud_hostname                  |cloud_ip         
|simzone_a               |5               |sut                     |simzone_a                       |151.234.121.242  
|simzone_b               |5               |sut                     |simzone_b                       |63.38.178.126    
|simzone_c               |5               |sut                     |simzone_c                       |209.93.27.5      
|simzone_d               |5               |lg                      |simzone_d                       |21.200.212.112   

The VMC object simzone_c, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (VMC object key)               |value                                
|update_attempts                          |312                                  
|update_frequency                         |12                                   

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |19.36M                        
|Redis Version                                        |2.6.2                         
|Uptime (in seconds)                                  |2512397                       
|Total Connections Received                           |185448                        
|Total Commands Processed                             |11289683                      
|Number of Keys                                       |272                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.5                         
|Storage Size                                         |18264064                      
|Data Size                                            |6141524                       
|Index Size                                           |1970416                       
|Average Object Size                                  |619.354981848                 
|Collections                                          |13                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |0                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |12                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |8                             
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |0                             
|VMs ARRIVED                                          |0                             
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |0                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |60                            

 ---------------- VMC-WIDE COUNTERS ----------------
|851FD80B-471A-5915-8AE1-DEFC6E0B82CF (simzone_b) VM RESERVATIONS|0                             
|893C8680-1F87-5029-9204-9C8BBE37D21C (simzone_a) VM RESERVATIONS|0                             
|322AE863-21B7-54F7-A0C4-7176A0E34BBF (simzone_d) VM RESERVATIONS|0                             
|DA279768-EC83-57F0-909E-7559192E2C87 (simzone_c) VM RESERVATIONS|0                             

